### PR TITLE
UI/UX Improvements

### PR DESCRIPTION
**Pull request by pabl-o-ce**
- navbar button show/hide
- new theme button
- code block fix width and theme
- new chat button on navbar with animations
- header chat stats of the settings on the conversational model

### CODE REVIEW

File: web/src/routes/chat/[id]/+page.svelte [hunk 1]
```diff
@@ -2,30 +2,47 @@
...
   import "highlight.js/styles/github-dark.css";
   import javascript from "highlight.js/lib/languages/javascript";
   import typescript from "highlight.js/lib/languages/typescript";
   import rust from "highlight.js/lib/languages/rust";
   import python from "highlight.js/lib/languages/python";
+  import html from "highlight.js/lib/languages/xml";
+  import css from "highlight.js/lib/languages/css";
+  import cpp from "highlight.js/lib/languages/cpp";
+  import powershell from "highlight.js/lib/languages/powershell";
+  import bash from "highlight.js/lib/languages/bash";
```
[the remark this answers is not in the document]
Comment (gaby) [reply web/src/routes/chat/[id]/+page.svelte:18]: @pabl-o-ce here

**Review by gaby — CHANGES_REQUESTED** [review]
See comments

File: web/src/routes/chat/[id]/+page.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (pabl-o-ce) [reply web/src/routes/chat/[id]/+page.svelte:18]: cool thanks

[the remark this answers is not in the document]
Comment (pabl-o-ce) [reply web/src/routes/chat/[id]/+page.svelte:18]: HTML is the same as xml

**Comment by pabl-o-ce** [comment]
https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md

**Comment by pabl-o-ce** [comment]
i add more sql, svelte, graphql

**Comment by gaby** [comment]
> i add more sql, svelte, graphql

Sounds good

**Comment by gaby** [comment]
@pabl-o-ce Add `toml` and `nginx`

**Comment by pabl-o-ce** [comment]
for svelte need to install [highlightjs-svelte](https://github.com/AlexxNB/highlightjs-svelte)
also terraform is a good one but also need to install [highlightjs-terraform](https://github.com/highlightjs/highlightjs-terraform)

additional packages

**Review by gaby — APPROVED** [review]

**Pull request merged**